compiler/src/iree/compiler/Codegen/SPIRV/test/lowering_matmul_fusion.mlir

```diff
@@ -0,0 +1,118 @@
+// RUN: iree-opt --split-input-file --pass-pipeline='builtin.module(hal.executable(hal.executable.variant(iree-spirv-lower-executable-target-pass)))' %s | FileCheck %s
+
+#compilation = #iree_codegen.compilation_info<
+    lowering_config  = <tile_sizes = [[32, 128, 1, 32]]>,
+    translation_info = <SPIRVMatmulPromoteVectorize pipeline_depth = 1>,
```

Comment:
It might be worth adding a test with non-trivial pipelining (depth >= 2 or depth = 1 + store stage 0) to make sure the fused producer doesn't interact poorly.